erpnext_documentation/www/docs/user/manual/en/automation/event_streaming.md

```diff
@@ -48,7 +64,14 @@ If you have some places where internet connectivity is low, for example, a store
 
 ## 3. Features
 
-### 3.1 Event Update Log
+
+### 3.1 Unsubscribe from the updates
+
+As an event consumer, if you wish to unsubscribe from the updates for any doctypes you had previously subscribed to, check unsubscribe against the doctype. You will not receive any more updates from the producer site for that particular doctype once you have unsubscribed.
+
+![Unsubscribe](/docs/assets/img/automation/unsubscribe-event.png)
+
+### 3.2 Event Update Log
 Event Update Log logs every create, update, and delete for documents that have consumers on the Event Producer site.
```

Comment:
"Event Update Log" logs every create, update, and delete action for documents that have consumers on the Event Producer site.